console/src/pages/console/settings/authentication/oauth/ConfigureGithubOauthButton.tsx

```diff
@@ -71,42 +74,45 @@ export function ConfigureGithubOAuthButton() {
       !data.logInWithGithub &&
       !getProjectResponse?.project?.logInWithEmail &&
       !getProjectResponse?.project?.logInWithPassword &&
-      !getProjectResponse?.project?.logInWithGoogle &&
-      !getProjectResponse?.project?.logInWithMicrosoft
+      !getProjectResponse?.project?.logInWithMicrosoft &&
+      !getProjectResponse?.project?.logInWithGithub
     ) {
       form.setError("logInWithGithub", {
         message:
-          "At least one of Log in with Email, Log in with Password, Log in with Google, Log in with Microsoft, or Log in with GitHub must be enabled.",
+          "At least one of Log in with Email, Log in with Password, Log in with GitHub, Log in with Microsoft, or Log in with GitHub must be enabled.",
       });
       return;
```

Comment:
IIUC, I think the previous logic was correct. Sanity check is if `project.logInWithGithub == true`, `project.logInWithX == false` and `data.logInWithGithub == false`, this check should fail, right? See also, the error message which now duplicates GitHub.

Reply:
This was an error I introduced, my bad.